voice/methods/calls/putCallsCallIdRecording.md

```diff
@@ -0,0 +1,28 @@
+{% method %}
+## Pause or resume recording on an active Call
+Pause or resume recording on an active phone call.
```

Comment:
This description is a little boring, any suggestions?

Reply:
seems pretty straight forward to me...